### DISCUSSION TITLE
Anybody else seeing black lines when resizing widescreen video using gpu-next on Windows with older Intel HD Graphics?

### DISCUSSION THREAD

**Discussion by Pentaphon** [mpv-player/mpv#14590] [General]
Windows 11. Only happens with gpu-next on older Intel HD graphics when resizing 16:9 or wider video. Only some sizes show these lines. Fullscreen is unaffected. 4:3 video is unaffected.

Not sure if it is worth opening an issue. 

![1](https://github.com/user-attachments/assets/afde3288-1454-4740-a587-23e2e2325aa4)
![3](https://github.com/user-attachments/assets/a6c16f37-f238-44f1-b25a-faeae4a42f4c)

**Comment by guidocella** [comment]
Duplicate of https://github.com/mpv-player/mpv/issues/13998

**Reply by Pentaphon** [reply]
> Duplicate of #13998

Ah, thank you, Closing.
